google/import_logging_project_sink_test.go

```diff
@@ -1,31 +0,0 @@
-package google
```

Comment:
I know it's slightly redundant but I'd prefer not removing this test file and just adding the import tests here.

Reply:
Done.